addons/l10n_ar/demo/account_customer_refund_demo.xml

```diff
@@ -6,7 +6,7 @@
         <field name="reason">Mercadería defectuosa</field>
         <field name="refund_method">refund</field>
         <field name="move_ids" eval="[(4, ref('demo_invoice_3'), 0)]"/>
-        <field name="journal_id" search="[('company_id', '=', ref('l10n_ar.company_ri')), ('type', '=', 'sale')]"/>
+        <field name="journal_id" model="account.move.reversal" eval="obj().env.ref('l10n_ar.demo_invoice_3').journal_id"/>
```

Comment:
As far as I remember we talk about the problem with the onchage and the compute of the default value for the jorunal_id was already fixed in 15.0 . So maybe we can remove the journal_id line directly from the demo data 
```suggestion
```